encrypt/encrypt.go

```diff
@@ -26,7 +26,12 @@ import (
 // Usage text that will be displayed as command line help text when using the
 // `help encrypt` command
 var Usage = `
-USAGE: %s encrypt -key <public-key-file> (-outdir <dir>) (-continue=true) [file(s)]
+USAGE: %s encrypt -key <public-key-file> (-target <target>) (-outdir <dir>) (-continue=true) [file(s)]
+
+The target can be one of the following:
+	bp.nbis.se
+	fega.nbis.se
+	gdc.nbis.se
```

Comment:
```suggestion
```

remove all references to specific sites

Reply:
There still should be some suggestion of how the target looks, or do you think that that information could be provided beforehand somewhere?

Reply:
The target is an URL, we should not give explicit examples.